.github/workflows/volume_group_operator_ci.yaml

```diff
@@ -96,4 +96,4 @@ jobs:
         with:
           name: images
           path: /tmp/images.txt
-          retention-days: 7
+          retention-days: 300
```

Comment:
why 300? IMO 100 is enough

Reply:
Barak suggested that I do it for 300 days if it is possible and does not affect anything.
but I got a warning in the build that I can do at max for 90 days. so I'm changed it.